crates/iroha_genesis/src/lib.rs

```diff
@@ -39,12 +41,13 @@ pub struct RawGenesisTransaction {
     chain: ChainId,
     /// Path to the [`Executor`] file
     executor: ExecutorPath,
+    /// Initial topology
+    topology: Vec<PeerId>,
     /// Parameters
     #[serde(skip_serializing_if = "Option::is_none")]
     parameters: Option<Parameters>,
     instructions: Vec<InstructionBox>,
-    /// Initial topology
-    topology: Vec<PeerId>,
+    wasm_triggers: Vec<GenesisWasmTrigger>,
```

Comment:
I'd also prefer to put it after `instructions` so that they can execute triggers but it's just not possible atm because triggers depend on some instructions executing before them